### ISSUE TITLE
Documentation confusing

### ISSUE THREAD

**Issue by mistraal** [chok/sfMelodyPlugin#54]
Please supply better examples on how to implement this plugin.

The readme you currently have is not complete. There is not enough explanation of each of the configuration settings.

A FULL example of how to start from scratch with step by step instructions is necessary, otherwise people do not know where to start.

**Comment by arianmaykon** [comment]
I agree, i manage to use this plugin but was a lot hard, even with the examples in the symfony site, and here on Github it was hard to use some of the features. Some things aren't clear enough

But i'm afraid the Symfony 1.x plugins will be receiving less attention as the 2.x version is out for a while now.
